README.md

```diff
@@ -25,7 +25,44 @@ uv add cicerone
 
 ## Quick Start
 
-```sh
-# Generate an object from an OpenAPI schema
-TODO
+```python
+from cicerone.spec import OpenAPISpec
+
+# Load from a file (auto-detects YAML/JSON)
+spec = OpenAPISpec.from_file("openapi.yaml")
+
+# Or from a URL
+spec = OpenAPISpec.from_url("https://api.example.com/openapi.json")
+
+# Or from a dict/JSON/YAML string
+spec = OpenAPISpec.from_dict({"openapi": "3.0.0", ...})
+spec = OpenAPISpec.from_json('{"openapi": "3.0.0", ...}')
+spec = OpenAPISpec.from_yaml('openapi: "3.0.0"\n...')
+
+# Traverse operations
```

Comment:
Show a more simple example with the top level module. Split this out with a brief one-sentence description of how you can use it once it is parsed.